webpack/ForemanTasks/Components/TaskDetails/TaskDetails.stories.js

```diff
@@ -1,5 +1,9 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
 import TaskDetails from './TaskDetails';
 
-storiesOf('TaskDetails', module).add('TaskDetails', () => <TaskDetails />);
+export default {
+  title: 'TaskDetails',
+  component: TaskDetails,
+};
+
+export const Basic = () => <TaskDetails />;
```

Comment:
why calling it `Basic`?

Reply:
I couldn't call it `TaskDetails` because it already exists.
`Basic` sounds like a good convention because the next one can be `WithSomething`, `WithSomethingElse`